docs/api/createAsyncThunk.mdx

```diff
@@ -202,7 +212,7 @@ type RejectedWithValue = <ThunkArg, RejectedValue>(
 
 To handle these actions in your reducers, reference the action creators in `createReducer` or `createSlice` using either the object key notation or the "builder callback" notation. (Note that if you use TypeScript, you [should use the "builder callback" notation to ensure the types are inferred correctly](../usage/usage-with-typescript.md#type-safety-with-extrareducers)):
 
-```js {2,6,14,23}
+```ts {2,6,14,23}
```

Comment:
Any other snippets changed to `ts` need the `no-transpile` as well - it's erroring because it can't find things like `createReducer`

Reply:
Thanks! I did let it crash on purpose, to see what was lacking. I've now pushed a new commit with all `ts` labels added with the `no-transpile`